teamcity.build

```diff
@@ -389,10 +389,10 @@
 				File.WriteAllText(outputFile, output);
 
 				if (brokenTests.Count > 0)
-					throw new Exception("Previously passing tests have been broken\n\n" + output);
+					throw new InvalidOperationException("Previously passing tests have been broken\n\n" + output);
```

Comment:
This is not InvalidOperationException. This is either need to stay as an `Exception` or another exception type, probably from Nant's namespace.

Reply:
Well, reverting. This script is currently no more used so changing it is not tested.